.gitattributes.example

```diff
@@ -0,0 +1,22 @@
+/.configurator  export-ignore
```

Comment:
The configurator is going to self-destroy after the set-up, so we can safely remove this line.